scripts/update_example_schemas.py

```diff
@@ -0,0 +1,55 @@
+import json
+from pathlib import Path
+from typing import Any, Dict, Union
+
+import pyarrow.parquet as pq
+
+
+def find_root_dir() -> Path:
+    """Find root geoparquet directory
+
+    This is a more robust approach than assuming __file__ refers to a specific place
+    """
+    folder = Path(__file__).resolve()
+
+    while folder.name != "geoparquet":
+        if folder == Path("/"):
+            raise ValueError("Could not find geoparquet folder")
+
+        folder = folder.parent
+
+    return folder
+
+
+def copy_parquet_schema_metadata_to_json(parquet_file: Path) -> None:
```

Comment:
I think it can be useful to have the opposite function too: read the JSON file and update the parquet file. This can speed up the updates while developing new versions of the spec.

Reply:
I agree that would be cool - would definitely help me ;) 

But let's just make an issue for this and revisit in the future. I think it'd be great to get this PR for 0.2.0.w

Reply:
+1 on making an issue and revisiting in the future

Reply:
Cool. Added #69